### PR TITLE
feat(errors): Add Sentry.captureException to ErrorHandler

### PR DESCRIPTION
**Pull request by leedongwei**
Adding Sentry to Sentry

### CODE REVIEW

File: src/sentry/static/sentry/app/utils/errorHandler.tsx [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
 
-    componentDidCatch(error, info) {
+    componentDidCatch(error: Error, info: {componentStack: string}) {
       // eslint-disable-next-line no-console
       console.error(
         'Component stack trace caught in <ErrorHandler />:',
         info.componentStack
       );
+
+      Sentry.captureException(error);
```
Comment (dashed) [inline src/sentry/static/sentry/app/utils/errorHandler.tsx]: @leedongwei This should already be emitted here https://github.com/getsentry/sentry/blob/37dc9190737f4a754b3c58447497a57b8a82ca55/src/sentry/static/sentry/app/views/routeError.jsx#L50

Comment (dashed) [reply src/sentry/static/sentry/app/utils/errorHandler.tsx]: Unless this does something else 🤔

Comment (billyvg) [reply src/sentry/static/sentry/app/utils/errorHandler.tsx]: Yeah I think @dashed is right

**Comment by leedongwei** [comment]
This PR was opened because the `errorHandler` was not surfacing thrown errors on development. Closing it because the issue has been identified to be unrelated to error boundaries on routes.

**Pull request closed without merging**